pkg/clusteragent/admission/mutate/auto_instrumentation.go

```diff
@@ -110,9 +112,78 @@ var (
 	}
 )
 
+// APMInstrumentation is the main handler for the APM instrumentation mutating webhook endpoints
+type APMInstrumentation struct {
```

Comment:
maybe we can name it `APMInstrumentationWebhook`

Reply:
Done :+1: